src/QuartoPlayerAgent.java

```diff
@@ -140,13 +134,14 @@ else if(state.value == curState.value) {
      */
     @Override
     protected String moveSelectionAlgorithm(int pieceID) {
-        QuartoGameState tmpState = new QuartoGameState(this.quartoBoard, Integer.MAX_VALUE, Integer.MIN_VALUE, true);
+        /*
+         * NOTE: the new game state is set as a minNode because then we pass this state to the
+         * generator to start trying to place the new piece we flip to a maxNode. This isn't
+         * intuitive to me but appears to result in a winning AI.
```

Comment:
YAY!!!... I'll look into this to see if we can have both an intuitive AI AND a winning AI